commit afccdf39d955c70dfcb7f71c4a10e7363cbb09f9 (refs/changes/00/2900/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-11-02T09:19:01-07:00 (1 year, 11 months ago)
    
    DOCKER-1123 docker build v1 image test should cleanup it's test container

diff --git a/test/integration/api-images-v1-v2.test.js b/test/integration/api-images-v1-v2.test.js
index 119673c..2cbbe72 100644
--- a/test/integration/api-images-v1-v2.test.js
+++ b/test/integration/api-images-v1-v2.test.js
@@ -388,7 +388,7 @@ test('test for error on docker v1 image build', function (tt) {
                 h.buildDockerContainer({
                     dockerClient: DOCKER_ALICE_HTTP,
                     params: {
-                        'rm': 'true'  // Remove container after it's built.
+                        'forcerm': 'true'  // Remove container after it's built.
                     },
                     test: t,
                     tarball: ctx.tarStream
